setup.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import io
+import os
+import re
+
+from setuptools import setup, find_packages
+
+RE_MD_CODE_BLOCK = re.compile(r'```(?P<language>\w+)?\n(?P<lines>.*?)```', re.S)
+RE_SELF_LINK = re.compile(r'\[(.*?)\]\[\]')
+RE_LINK_TO_URL = re.compile(r'\[(?P<text>.*?)\]\((?P<url>.*?)\)')
+RE_LINK_TO_REF = re.compile(r'\[(?P<text>.*?)\]\[(?P<ref>.*?)\]')
+RE_LINK_REF = re.compile(r'^\[(?P<key>[^!].*?)\]:\s*(?P<url>.*)$', re.M)
+RE_BADGE = re.compile(r'^\[\!\[(?P<text>.*?)\]\[(?P<badge>.*?)\]\]\[(?P<target>.*?)\]$', re.M)
+RE_TITLE = re.compile(r'^(?P<level>#+)\s*(?P<title>.*)$', re.M)
+
+BADGES_TO_KEEP = ['gitter-badge']
+
+RST_TITLE_LEVELS = ['=', '-', '*']
+
+RST_BADGE = '''\
+.. image:: {badge}
+    :target: {target}
+    :alt: {text}
+'''
+
+
+def md2pypi(filename):
```

Comment:
Maybe it's time to start publishing markdown description ? ;)

Reply:
I'll let you strike with the first PR ;-)